### PR TITLE
Show Group Status through the favicon

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by eliperelman** [comment]
No white, but I'd say a gap, that way it is transparent. I can help edit the image if you need.

**Comment by helfi92** [comment]
👀 @djmitche @eliperelman 

screen shot 2017-10-25 at 4 39 45 pm

**Comment by helfi92** [comment]
I'll send a different PR to remove the pop notification system we have after this gets merged.

**Comment by djmitche** [comment]
I like :)

File: src/views/UnifiedInspector/GroupProgress.js [hunk 1]
```diff
@@ -96,6 +97,20 @@ export default class GroupProgress extends React.PureComponent {
     );
   }
 
+  handleGroupStateChange(percents) {
```
Comment (eliperelman) [inline src/views/UnifiedInspector/GroupProgress.js]:
```js
handleGroupStateChange(percents) {
  if (percents.completed === 100) {
    this.props.onGroupStateChange('success');
  } else if (percents.failed) {
    this.props.onGroupStateChange('failed');
  } else {
    this.props.onGroupStateChange('pending');
  }
}
```

File: src/views/UnifiedInspector/Inspector.js [hunk 2]
```diff
@@ -483,6 +491,24 @@ export default class Inspector extends React.PureComponent {
     return artifacts.filter(({ name }) => /^public\/logs\//.test(name));
   }
 
+  getFavicon = () => {
```
Comment (eliperelman) [inline src/views/UnifiedInspector/Inspector.js]: I don't think this needs to be an arrow function.

File: src/views/UnifiedInspector/Inspector.js [hunk 3]
```diff
@@ -483,6 +491,24 @@ export default class Inspector extends React.PureComponent {
...
+    let FavIcon;
```
Comment (eliperelman) [inline src/views/UnifiedInspector/Inspector.js]:
```js
getFavicon() {
  switch (this.state.groupState) {
    case 'pending':
      return pendingFavIcon;
    case 'failed':
      return failedFavIcon;
    default:
      return successFavIcon;
  }
}
```

**Pull request merged**